src/content/get-started/advanced-commands-and-concepts.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Advanced Commands & Concepts for Developers
+description: Introduction to development concepts within Okteto
+sidebar_label: Advanced Commands and Concepts
+id: advanced-commands-and-concepts
+---
+
+import Image from '@theme/Image';
+
+## Advanced CLI Commands
+
+| Command          | Description                                                                                                                 |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| okteto namespace | Allows you to deal with namespaces (create, delete, use).                                                                   |
+| okteto exec      | Allows you to execute a command in your development container. `okteto exec [devName] -- COMMAND`                           |
+| okteto destroy   | Destroys a currently deployed Development Environment                                                                       |
+| okteto logs      | Used to view the logs of a Development Environment                                                                          |
+| okteto endpoints | Used to view your application’s HTTPS URLs automatically generated by Okteto                                                |
+| okteto doctor    | Generates a [doctor file](reference/okteto-cli.mdx#doctor) with the okteto logs for a given development container           |
+
+### Commonly Used Flags
+
+- **`okteto deploy --dependencies`**: Installs dependencies defined in your okteto manifest
+- **`okteto deploy --build`**: Forces a rebuild of the images, incorporating any local changes
+- **`okteto build --no-cache`**: Do not use a cache when building an image
+- `okteto destroy --volumes`: Destroy the persistent volumes created by the development environment. Persistent volumes are a storage resource that persists beyond the lifecycle of the Development Environment. This allows data to survive and be accessible by new Development Applications that may be created later. This could be useful for data for databases or application state while developing.
+- `okteto destroy --all`: Destroy all Development Environments, including resources annotated with `dev.okteto.com/policy: keep`
+
+## Important Concepts
```

Comment:
If this doc is about "Advanced" concepts, I don't think important concepts go here